### PR TITLE
http: Use Content-Type application/octet-stream

### PR DESCRIPTION
**Pull request by abhinav**
CC @yarpc/golang @yarpc/yarpc

### CODE REVIEW

**Comment by breerly** [comment]
Title seems wrong. Code is :shipit:

**Comment by abhinav** [comment]
Oh, weird.

File: transport/http/outbound.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ func (o outbound) Call(ctx context.Context, req *transport.Request) (*transport.
 	request.Header.Set(ServiceHeader, req.Service)
 	request.Header.Set(ProcedureHeader, req.Procedure)
 	request.Header.Set(TTLMSHeader, fmt.Sprintf("%d", req.TTL/time.Millisecond))
+	request.Header.Set("Content-Type", "application/octet-stream")
```
Comment (prashantv) [inline transport/http/outbound.go]: always? even for json?

**Comment by abhinav** [comment]
Turns out more changes are required. JSON should use application/json and we're discussing if thrift should use something else too. Abandoning this diff.

**Pull request closed without merging**